internal/authorization/authorization.go

```diff
@@ -14,6 +14,10 @@ const (
 	isAuthorizedCost = 10
 )
 
+type Auth interface {
```

Comment:
Couldn't we just have one eg `PVMInvoker` interface that's implemented by both auth and refine? And then this interface would just live in the results package.

Reply:
(also given this mostly seems in place for mocking)

Reply:
yes you are right! I moved them